docker/nginx/nginx.conf

```diff
@@ -71,7 +71,9 @@ server {
     }
 
     location /screenly_assets {
```

Comment:
This doesn't look right. IIRC this endpoint should only be accessible from the local device.

Reply:
Given that it's a constraint that we should consider, then I should focus on editing the C++ code instead and not exposing the asset files in NGINX.

Reply:
I don't think any C++ code edits are needed. You just need to introspect and analyze how the traffic is flowing and perhaps provide a slightly wider policy to cover all possible Docker IPs.

Reply:
Thanks. After investigating thoroughly, it appears that the D-Bus messages are going through.